website/content/en/guides/level-up/vector-tap-guide.md

```diff
@@ -0,0 +1,243 @@
+---
+title: Using Vector `tap`
+short: Vector tap
+description: Learn how to use the Vector `tap` CLI command to examine events as they flow through your pipeline and troubleshoot issues.
+author_github: https://github.com/001wwang
+domain: operations
+weight: 6
+tags: ["cli", "tap", "level up", "guides", "guide"]
+---
+
+{{< requirement >}}
+Before you begin, this guide assumes the following:
+
+* You understand the [basic Vector concepts][concepts]
+* You understand [how to set up a basic pipeline][pipeline]
+
+[concepts]: /docs/about/concepts
+[pipeline]: /docs/setup/quickstart
+{{< /requirement >}}
+
+Vector's `tap` CLI command allows you to observe events as they flow to and from
+components in your pipelines. If you've ever attached a `console` sink to an
+output for debug purposes and wondered if there's a better way, you're in the
+right place. This guide walks you through how `vector tap` can be used to level
+up your troubleshooting experience.
+
+## Getting Started
+
+To start, we'll reference the following base configuration, but feel free to
+substitute your own. Just note that the [Vector API] must be enabled for `vector
+tap` to work. See [under the hood](#under-the-hood) for more details.
+
+```toml
+[api]
+enabled = true
+
+[sources.in]
+type = "demo_logs"
+format = "shuffle"
+lines = [
+  "test1",
+  "test2",
+]
+
+[sinks.out]
+type = "blackhole"
+inputs = ["in*"]
+```
+
+Run Vector with this configuration and watch the configuration for changes.
+
+```console
+vector --config {config} -w
```

Comment:
Minor nit:

```suggestion
vector --config path/to/config.toml -w
```

This is likely obvious to the majority of users, but I could imagine someone being confused by seeing `{config}` and not realising that we're asking them to put the config snippet into a _file_ and reference it by path.